### DISCUSSION TITLE
How to run a program in powershell instead of CMD

### DISCUSSION THREAD

**Discussion by zirnc** [microsoft/vscode-cmake-tools#2340] [Q&A] [answered]
![cmd](https://user-images.githubusercontent.com/42088872/150907755-4ca303ff-9114-4fe4-8fc1-e158eb422ff8.gif)

The default profile is powershell.
![image](https://user-images.githubusercontent.com/42088872/150907963-c0918436-26ca-4a15-9a1b-58bfa76b4471.png)

**Comment by bobbrow** [comment, marked as answer]
This was fixed in https://github.com/microsoft/vscode-cmake-tools/pull/2311 which released with version 1.10.4

**Reply by zirnc** [reply]
thx
